index.html

```diff
@@ -25,8 +22,8 @@ <h2>Question</h2>
           impedit debitis odio asperiores iusto aliquid ipsam cum facere amet
           aliquam illum quidem et!
         </p>
-        <button class="button">Button text</button>
-        <p class="answer">
+        <button data-js="button1" class="button">KLICKMICH</button>
```

Comment:
You could write this lowercase and use the CSS property [text-transform: uppercase](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).